### ISSUE TITLE
run proxy by default when adding client without NSM support

### ISSUE THREAD

**Issue by grammoboy2** [Houston4444/RaySession#173]
Am I right that RaySession launches the proxy client by default when a application without NSM support is added? 

I can imagine this would speed up the loading of the sessions, cause the server doesn't have to wait for a announce by the 'dumb' client, it gets the announce from the proxy client. 

In non-session-manager and in agordejo it is possible to launch applications without NSM support directly, that is, without using NSM-Proxy. Then the server waits +- 5 sec for a announce, which slows down the loading of sessions quite a bit.

**Comment by Houston4444** [comment]
> Am I right that RaySession launches the proxy client by default when a application without NSM support is added?

Not exactly, when you launch executable unchecking NSM checkbox, it launches the program with Ray-Hack pseudo-protocol. It is launched directly by the daemon, and estimate the program as ready after 500ms (if CONFIG_FILE is set). There is also ray-proxy executable, it was the previous way, but it is old and unmaintained, I will probably announce at next release that I will remove it.

> In non-session-manager and in agordejo it is possible to launch applications without NSM support directly, that is, without using NSM-Proxy. Then the server waits +- 5 sec for a announce, which slows down the loading of sessions quite a bit.

RaySession remembers which executables never send any message and reduce the waiting time for them.
